app/stacks/uk-west/common/data.tf

```diff
@@ -4,3 +4,13 @@ data "azurerm_key_vault_secret" "applications_service_vpn_gateway_shared_key" {
   name         = azurerm_key_vault_secret.applications_service_vpn_gateway_shared_key.name
   key_vault_id = azurerm_key_vault.environment_key_vault.id
 }
+
+data "azurerm_key_vault_secret" "back_office_sql_server_password" {
```

Comment:
Also thought on this... Does SQL server actually pull the values fro Key Vault? Or are these set up once when it's created? If the latter then we don't need data sources as we just want to set the username/password and store it in a secret so we have something to reference

Reply:
In effect it does, via the data resources that pull the values of those secrets and pass them securely via outputs and input variables to the Back Office stack. The latest version of each secret is read. However, in thinking about it the data resource could be moved to the Back Office stack and retrieved that, rather than being passed through in variables.

Creating the password via the `random_password` resource then storing that value in Key Vault for reference may be smoother, as you suggest.

Reply:
Resolved in https://github.com/Planning-Inspectorate/infrastructure-environments/pull/60/commits/c654cf28a9584b639ac7152fd10796f0d493b66e